packages/rrweb/src/record/observers/canvas/canvas-manager.ts

```diff
@@ -118,16 +118,20 @@ export class CanvasManager implements CanvasManagerInterface {
         unblockSelector,
       );
     if (recordCanvas && typeof sampling === 'number')
-      this.initCanvasFPSObserver(
-        sampling,
-        win,
-        blockClass,
-        blockSelector,
-        unblockSelector,
-        {
-          dataURLOptions,
-        },
-      );
+      try {
+        this.initCanvasFPSObserver(
+          sampling,
+          win,
+          blockClass,
+          blockSelector,
+          unblockSelector,
+          {
+            dataURLOptions,
+          },
+        );
+      } catch {
+        // Error when initializing canvas...
```

Comment:
Should this bubble up to error handler?

Reply:
good point, I'll wrap this in callbackWrapper instead!